libraries/AP_HAL_Linux/RCInput.h

```diff
@@ -3,6 +3,7 @@
 #include "AP_HAL_Linux.h"
 
 #define LINUX_RC_INPUT_NUM_CHANNELS 16
+#define MIN_NUM_CHANNELS 5
```

Comment:
There's no reason to make this define public and putting it here means that anyone including this header would have this non-namespaced define. Please move the define to the .cpp rather than the .h